modules/Db2/outputs.tf

```diff
@@ -0,0 +1,34 @@
+output "db2_project_name" {
+  value = var.db2_project_name
+  description = "The namespace/project for Db2"
+}
+
+output "db2_admin_user_password" {
+  value = var.db2_admin_user_password
+  description = "Db2 admin user password defined in LDAP"
+}
+
+output "db2_admin_username" {
+  value = var.db2_admin_username
+  description = "Db2 admin username defined in LDAP"
+}
+
+//output "db2_host_name" {
```

Comment:
Why are these outputs commented out?  Doesn't the CP4BA module need these values?